Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.py

```diff
@@ -0,0 +1,832 @@
+
+    return argument_value
+
+
+def get_time_parameter(arg: Optional[str]):
+    """
+    parses arg into date time object with aware time zone if 'arg' exists.
+    If no time zone is given, sets timezone to UTC.
+    Returns the date time object created.
+    Args:
+        arg (str): The argument to turn into aware date time.
+
+    Returns:
+        - (None) If 'arg' is None, returns None.
+        - (datetime): If 'arg' is exists, returns date time.
+    """
+    maybe_unaware_date = arg_to_datetime(arg, is_utc=True)
+    if not maybe_unaware_date:
+        return None
+
+    aware_time_date = maybe_unaware_date if maybe_unaware_date.tzinfo else UTC_TIMEZONE.localize(
+        maybe_unaware_date)
+
+    return aware_time_date
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def fetch_incidents_command(client: Client, params: Dict, last_run: Dict):
+    """
+    Fetches new incidents from FraudWatch.
+    Because of FraudWatch limitation, such as 'from_date' being accurate only by day, and incidents returned
+    are not ordered by their date opened.
+    1) First_fetch time is limited to 1 day.
+    2) Every fetch_incidents call, we will fetch all the pages from the day of the time stamp saved in last run.
+    3) Filter all incidents that their date opened was earlier than last saved time stamp.
+    4) Sort the remained incidents by their date opened.
+    5) Return the first 'max_fetch' incidents in the sorted list.
+    Limiting first_fetch is needed because we must traverse all incidents that might be relevant, therefore the
+    list cannot be very large.
+    Uses Demisto last run parameters in the following way:
+    - 'last_fetch_time' - The latest time fetching was done. Keeps track of incidents to be fetched by their date.
+
+    Args:
+        client (Client): FraudWatch client to perform API call to fetch incidents.
+        params (Dict): Demisto params.
+        last_run (Dict): Demisto last run.
+
+    Returns:
+        (List[Dict], Dict): Incidents and new run parameters.
+    """
+    brand = params.get('brand')
+    status = params.get('status')
+    limit = arg_to_number(params.get('max_fetch', DEFAULT_LIMIT_VALUE))
+
+    yesterday = get_time_parameter('1 day')
+    first_fetch_time_string = params.get('first_fetch', '1 days').strip()
+    minimum_date_opened_str = last_run.get('last_fetch_time', first_fetch_time_string)
+    # Between yesterday date and last fetch date we choose the latest
+    minimum_date_opened = max(get_time_parameter(minimum_date_opened_str), yesterday)
+
+    from_date = minimum_date_opened.strftime(FRAUD_WATCH_DATE_FORMAT)
+    incidents = []
+
+    # Need to fetch all incidents because FraudWatch does not sort incidents based on dates.
+    current_page = 1
+    while True:
+        raw_response = client.incidents_list(brand=brand, status=status, page=current_page, limit=MAX_LIMIT_VALUE,
+                                             from_date=from_date, to_date=None)
```

Comment:
Due to my previous comment this will not be needed
```suggestion
                                             from_date=from_date)
```